doorpi.py

```diff
@@ -275,14 +278,14 @@ def handle_open(cls):
 
         # TODO: put the on-sleep-off into a thread to mitigate impact on website delivery
         if not emulation:
-            DoorSocketHandler.door.on()
+            DoorSocketHandler.door.gpioOpenPin.on()
```

Comment:
I'd like to see a Door.open() that threaded opens the door ;-)

Reply:
sure, good idea